### ISSUE TITLE
How to use the new UI?

### ISSUE THREAD

**Issue by tregismoreira** [askmike/gekko#624]
Hey guys, I'm trying to use the new user interface for gekko, but I couldn't find how. Can you help me? Thanks in advance.

**Comment by askmike** [comment]
First you have to download the develop branch over [here](https://github.com/askmike/gekko/tree/develop). After that, follow these instructions for installing and running the UI:

https://github.com/askmike/gekko/blob/develop/docs/installing_gekko.md

**Comment by tregismoreira** [comment]
Thanks! I was trying to use the branch `origin/ui/market-watchers`.

**Comment by askmike** [comment]
I am trying to get the UI into stable asap (see #611 for details). For that I merged ui/market-watchers into develop just so I can get more feedback. Note that the UI is very basic and only backtesting is fully integrated at this moment.

**Comment by nedievas** [comment]
Can not get to web ui:
[bitfinex@gene gekko-develop]$ node gekko --ui
```

    ______   ________  __    __  __    __   ______
   /      \ /        |/  |  /  |/  |  /  | /      \
  /$$$$$$  |$$$$$$$$/ $$ | /$$/ $$ | /$$/ /$$$$$$  |
  $$ | _$$/ $$ |__    $$ |/$$/  $$ |/$$/  $$ |  $$ |
  $$ |/    |$$    |   $$  $$<   $$  $$<   $$ |  $$ |
  $$ |$$$$ |$$$$$/    $$$$$  \  $$$$$  \  $$ |  $$ |
  $$ \__$$ |$$ |_____ $$ |$$  \ $$ |$$  \ $$ \__$$ |
  $$    $$/ $$       |$$ | $$  |$$ | $$  |$$    $$/
   $$$$$$/  $$$$$$$$/ $$/   $$/ $$/   $$/  $$$$$$/

        Gekko v0.4.4
        I'm gonna make you rich, Bud Fox.


Serving Gekko UI on http://localhost:3000/


  <-- GET /favicon.ico
  --> GET /favicon.ico 404 3ms -
```
and I get this message in browser:

```
Disconnected

Something happened to either Gekko or the connection. Please check the terminal where Gekko is running or your network connection.
```

**Comment by nedievas** [comment]
```
[bitfinex@gene gekko-develop]$ cp sample-config.js config.js
[bitfinex@gene gekko-develop]$ node gekko

    ______   ________  __    __  __    __   ______
   /      \ /        |/  |  /  |/  |  /  | /      \
  /$$$$$$  |$$$$$$$$/ $$ | /$$/ $$ | /$$/ /$$$$$$  |
  $$ | _$$/ $$ |__    $$ |/$$/  $$ |/$$/  $$ |  $$ |
  $$ |/    |$$    |   $$  $$<   $$  $$<   $$ |  $$ |
  $$ |$$$$ |$$$$$/    $$$$$  \  $$$$$  \  $$ |  $$ |
  $$ \__$$ |$$ |_____ $$ |$$  \ $$ |$$  \ $$ \__$$ |
  $$    $$/ $$       |$$ | $$  |$$ | $$  |$$    $$/
   $$$$$$/  $$$$$$$$/ $$/   $$/ $$/   $$/  $$$$$$/

        Gekko v0.4.4
        I'm gonna make you rich, Bud Fox.



 ERROR: Please specify a config file.
```

**Comment by askmike** [comment]
@nedievas please create your own issue.

> Can not get to web ui:
>> Something happened to either Gekko or the connection. Please check the terminal where Gekko is running or your network connection.

Strange.. What OS and what browser are you using?

>  ERROR: Please specify a config file.

In the develop branch the UI is the default interface for gekko. You can still use the command line (considered for "advanced" users) but have to do everything yourself: specifying a config, setting up all plugins to serve your purpose, etc.

    node gekko --config config.js

**Comment by Schlitten** [comment]
after I figured out how to use the stable version and was quite smooth with backtesting I tried to get the ui working.
downloaded the develop zip, installed dependencies(npm) and started ui. So far everything worked fine.

However, importing data doesn't work and if I start backtesting (imported history data from stable version) I get the graph but no buy and sell indicators:
![screen shot 2017-03-17 at 16 01 32](https://cloud.githubusercontent.com/assets/26486861/24049034/1333f310-0b2b-11e7-82d5-90f9023d8bdb.png)

the console delivers also strange output:
![screen shot 2017-03-17 at 16 04 43](https://cloud.githubusercontent.com/assets/26486861/24049169/80fac004-0b2b-11e7-8541-8aba853a6774.png)

anything I missed out? MacOS

Thanks!
